### DISCUSSION TITLE
[Help]: How can i make Vertical slider autoplay in opposite direction

### DISCUSSION THREAD

**Discussion by Pixsal** [davidjerleke/embla-carousel#949] [Help] [answered]
### Summary

So for horizontal slider we can use RTL so it autoplays / autoscrolls in opposite direction.

But how can i make vertical slider which always slides from bottom to top in opposite direction.

I want slider to scroll from top to bottom instead. I can't seem to find a solution for this, Any help will be greatly appreciated.

### If applicable, which variants of Embla Carousel are relevant to this question?

- [X] embla-carousel **(Core)**
- [ ] embla-carousel-react
- [ ] embla-carousel-vue
- [ ] embla-carousel-svelte
- [X] embla-carousel-autoplay
- [X] embla-carousel-auto-scroll
- [ ] embla-carousel-solid
- [ ] embla-carousel-auto-height
- [ ] embla-carousel-class-names
- [ ] embla-carousel-fade
- [ ] embla-carousel-docs **(Documentation)**
- [ ] embla-carousel-docs **(Generator)**

**Comment by davidjerleke** [comment]
Hi @Pixsal,

Thanks for your question. Please clarify if you are using either `embla-carousel-autoplay` **or**
 `embla-carousel-auto-scroll`? Both can’t be active at the same time.

**Comment by Pixsal** [comment]
I am using auto-scroll, But i would also like to use autoplay in the future for the similar effect. Thanks for good work @davidjerleke so far the slider has been a breeze to work with.

**Comment by Pixsal** [comment, marked as answer]
I figured out how to achieve it, I am using scrollPrev instead of scrollNext seems to do the trick.

**Reply by davidjerleke** [reply]
@Pixsal so you solved the problem then? Or did you have additional questions?

**Reply by davidjerleke** [reply]
@Pixsal did you see my [previous comment](https://github.com/davidjerleke/embla-carousel/discussions/949#discussioncomment-10180711)?

**Reply by Pixsal** [reply]
Its Resolved david.
